gnocchi/storage/__init__.py

```diff
@@ -276,8 +276,9 @@ def _version_check(name, v):
         """
         return name.split("_")[-1] == 'v%s' % v
 
-    def get_aggregated_measures(self, metrics_and_aggregations,
-                                from_timestamp=None, to_timestamp=None):
+    def get_aggregated_measures(self, metric, aggregations,
```

Comment:
Can we keep the previous signature that allows to pass multiple metrics too.

Reply:
Yes this is meant to be batched :)